d2l-icon-button.html

```diff
@@ -18,7 +18,8 @@
 			behaviors: [window.D2L.PolymerBehaviors.IconButtonBehavior],
```

Comment:
Yeah we should just pick an approach and stick with it. We either take the text content (which should be required) and apply it to the title or vice-versa. I mentioned my preference for using the text content earlier, simply because it will be more common and familiar to people using `<button>`s.